app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -68,6 +80,18 @@ private fun OngoingCallContent(ongoingCallViewModel: OngoingCallViewModel) {
             verticalArrangement = Arrangement.Center,
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
+            if (ongoingCallViewModel.callEstablishedState.isCameraOn) {
+                //TODO fix memory leak when the app goes to background with video turned on
+                // https://issuetracker.google.com/issues/198012639
+                // The issue is marked as fixed in the issue tracker,
+                // but we are still getting it with our current compose version 1.2.0-beta01
+                AndroidView(factory = {
+                    val videoPreview = VideoPreview(it)
+                    ongoingCallViewModel.setVideoPreview(null)
```

Comment:
why are we setting it here first to null ?

Reply:
To be safe, we could have an older connected view